src/pluginlaunch.c

```diff
@@ -137,7 +141,17 @@ update_running_processes (kb_t kb)
                     g_message ("%s (pid %d) is slow to finish - killing it",
                                oid, processes[i].pid);
                   if (prefs_get_bool ("advanced_log"))
-                    kb_item_add_str (kb, "log/timedout", oid, 0);
+                    {
+                      char buffer[2048];
+
+                      snprintf (buffer, sizeof (buffer),
+                                "LOG||| |||general/Host_Details||| |||<host>"
+                                "<detail><name>nvt_end_time</name><value>>"
```

Comment:
``<value>>`` -> ``<value>``

Reply:
@cfi-gb Thanks for spotting it. PR updated.